README.md

```diff
@@ -36,11 +35,17 @@ You can now claim the initial NEO and GAS:
 
 `./create_wallet` will display several internal error messages, which is expected as long as at the end you still get a success message.
 
+If you call `./create_wallet.sh` or `./docker_run_and_create_wallet.sh`, it will create 2 files in your current directory :
+
+- `neo-privnet.wallet`: a wallet you can use with neo-python 
+- `neo-privnet.wif`: a wif private key you can import into other clients (neo-gui for example).
+ 
+Those files will get you access to the wallet containing all the NEOs of your private network.
```

Comment:
"all the NEO and GAS for your private network" please

Reply:
Technically, the printed output is :
Finished, Ae9MUYt44Le4ux2Hcd8JEnDpfacN7xQCgD should now own all the NEO on the private network.

Then it claims 48 Gas :
Claiming the GAS...

---------------------------------------------------------------
Will make claim for 48.0 GAS
------------------------------------------------------------------